src/core/src/op/strided_slice.cpp

```diff
@@ -237,27 +238,54 @@ bool op::v1::StridedSlice::has_evaluate() const {
     return get_input_size() == 4;
 }
 
-namespace {
-bool strided_slice_input_check(const ov::Node* node) {
-    if (!node->get_input_tensor(1).has_and_set_bound() || !node->get_input_tensor(2).has_and_set_bound() ||
-        !node->get_input_tensor(3).has_and_set_bound())
+bool op::v1::StridedSlice::indicies_input_has_and_set_bounds(const size_t port,
+                                                             const std::vector<int64_t>& masks) const {
+    const auto& lb_t = get_input_tensor(port).get_lower_value();
+    const auto& ub_t = get_input_tensor(port).get_upper_value();
+
+    if (lb_t && ub_t) {
+        using TCast = int64_t;
+        constexpr auto i64_cast = ov::util::Cast<TCast>();
+        const auto lb = ov::get_tensor_data_as<TCast>(lb_t, i64_cast);
+        const auto ub = ov::get_tensor_data_as<TCast>(ub_t, i64_cast);
+        const auto mask_set = convert_mask_to_axis_set(masks);
+
+        size_t axis = 0;
+        return std::equal(lb.cbegin(), lb.cend(), ub.cbegin(), [&axis, &mask_set](TCast lhs, TCast rhs) -> bool {
```

Comment:
should it be const references instead?
```suggestion
        return std::equal(lb.cbegin(), lb.cend(), ub.cbegin(), [&axis, &mask_set](const TCast& lhs, const TCast& rhs) -> bool {
```

Reply:
it's ok for int64_t type, ref is not needed

Reply:
Using reference for fundamental types can cause indirect access which can be slower than direct (by value) access. And pass fundamental type by value shouldn't be issue.